### DISCUSSION TITLE
alter table

### DISCUSSION THREAD

**Discussion by smrn90** [adonisjs/core#1926] [Help]
I have this test_users table with some data already. I want to add a column 'isVerified' which would be boolean and set to false as default.
And i want to alter this column 'phone_number' as unique. 

```
class TestuserSchema extends Schema {
  up () {
    this.create('test_users', (table) => {
      table.increments('id')
      table.string('email', 254).notNullable().unique()
      table.string('password', 255).notNullable()
      table.string('phone_number').notNullable()
      table.integer('age')
      table.string('gender', 15)
      table.integer('userType_id')
      table.datetime('registered_on')
      table.string('profilePic_url')
      table.string('username', 80).notNullable().unique()
      table.string('address', 255)
      table.string('createdBy', 15)
      table.string('modifiedBy', 15)
      table.string('row_state')
      table.string('row_version')
      table.timestamps()
    })
  }
```
 I read that i have to create this migration test_user again choosing select table option and alter it, but i couldnt.

i did adonis make:migration test_user and tried doing this.  
```
class TestUserSchema extends Schema {
  up () {
    this.alter('test_users', (table) => {
      // alter table
      table.boolean('isVerified').defaultTo(false)
      table.string('phone_number').notNullable().unique()
    })
  }
```

 ```
 down () {
    this.alter('test_users', (table) => {
      // reverse alternations
      table.boolean('isVerified').defaultTo(false)
      table.string('phone_number').notNullable().unique()
    })
  }
}
```

The error is 
![error](https://user-images.githubusercontent.com/72269964/99517180-ecf5a000-2943-11eb-882e-0b666bf644c9.png)

Any help would be appreciated.

**Comment by thetutlage** [comment]
Please format the code blocks. Here's link to Github markdown guide https://guides.github.com/features/mastering-markdown/

**Reply by smrn90** [reply]
is it better?

**Reply by thetutlage** [reply]
I think it has been answered in your other thread. https://github.com/adonisjs/core/discussions/1927
